modules/ROOT/pages/create-an-api-configure-service-mesh.adoc

```diff
@@ -21,4 +42,92 @@ The Endpoint Configuration window is displayed.
 . In the *Application type* field, select the *Non-Mule application* radio button.
 . Optionally, specify the *Implementation URI*, and click *Save*.
 +
-Your API is now created.
\ No newline at end of file
+Your API is now created.
+
+[[create-api-ASM-CLI]]
+== Creating an API Using Anypoint Service Mesh CLI
+
+Before you create the API, you must complete the task prerequisite.
+
+*Task Prerequisite*
+
+Ensure that you obtain the environment ID and organization ID. You can perform this task from either API Manager or from Access Management. You require the <INSERT_PERMISSION> permission or <INSERT_ROLE> role to see to obtain these credentials.
```

Comment:
We need to add the permissions part of who can create the API. I added a paragraph before on the Doc Changes after GA. Basically, to create the API you need `In order to create an API you will need to have at least Exchange Contributor for the Business group where you are creating the asset and Manage APIs configuration for the environment where you are creating the API Instance. `

Reply:
But keep in mind that that is different from the permission `Ensure that you obtain the environment ID and organization ID.`

Reply:
For the latter permission is just access to APIM.

Reply:
Done! Thanks!